### PR TITLE
Create README-PT.md

### PR DESCRIPTION
**Pull request by KinhaChaves**
Translation docs

### CODE REVIEW

File: README-PT.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+[![Build Status](https://travis-ci.org/malukenho/MigrateDB.png?branch=master)](https://travis-ci.org/malukenho/MigrateDB) [![Latest Stable Version](https://poser.pugx.org/malukenho/migratedb/v/stable.png)](https://packagist.org/packages/malukenho/migratedb) [![Total Downloads](https://poser.pugx.org/malukenho/migratedb/downloads.png)](https://packagist.org/packages/malukenho/migratedb) [![Latest Unstable Version](https://poser.pugx.org/malukenho/migratedb/v/unstable.png)](https://packagist.org/packages/malukenho/migratedb) [![License](https://poser.pugx.org/malukenho/migratedb/license.png)](https://packagist.org/packages/malukenho/migratedb)
+# MigrateDB
```
Comment (malukenho) [inline README-PT.md:2]: Put a space here.

File: README-PT.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+
+
+
```
Comment (malukenho) [inline README-PT.md:6]: Overdose of spaces here.

**Comment by Ocramius** [comment]
Not sure if maintaining translations is a good idea, unless you completely take over that part.

File: README-PT.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+**MigrateDB** é uma ferramenta simples para migrar dados entre bancos de dados.
+
+#### 1ª Etapa:
+
+Instalando o  **MigrateDB** fica muito simples usar o composer :3
```
Comment (malukenho) [inline README-PT.md:11]: You have much spaces after `o` and your phrase have no much sense. I think you mean:

>  É muito simples instalar o **MigrateDB** utilizando o composer :3

or anything else.

**Comment by malukenho** [comment]
@Ocramius So do you think it's can be closed?

**Comment by Ocramius** [comment]
@malukenho this is up to you. I generally assume that English is a minimum
requirement for software developers.

That said, if @KinhaChaves can keep the docs in sync over time, then it is
ok to merge, but it needs commitment, or the file may be dropped in future.

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by malukenho** [comment]
As talked with @KinhaChaves on chat, this issue can be closed.

**Pull request closed without merging**